src/components/Avatars/AvatarBase/__test__/AvatarBase.test.tsx

```diff
@@ -0,0 +1,51 @@
+import AvatarBase, { AvatarBaseProps } from "../";
+
+import { render, screen } from "@testing-library/react";
+
+describe("<AvatarBase />", () => {
+    let props: AvatarBaseProps;
+
+    it("should render a 36px div inside a div containing an img", () => {
+        props = {
+            src: "/fake.png",
+            alt: "a fake image",
+        };
+        const { container } = render(<AvatarBase {...props} />);
+        const box = container.querySelector("div");
+        const avatar = box!.querySelector("div");
+        const img = avatar!.querySelector("img");
+        expect(box).toBeDefined();
+        expect(avatar).toBeDefined();
+        expect(img).toBeDefined();

```

Comment:
If you want to get serious on testing, feel free to research on how to use "data-testid"
This method is fine right now but for bigger components it's not usable so I prefer not to use it